src/components/atoms/Plugin/PluginIFrame/useIFrame.ts

```diff
@@ -61,6 +61,10 @@ export default function useIFrame({
     [iframeCanBeVisible, onRender, postMessage],
   );
 
+  useEffect(() => {
+    setIFrameState(state => [state[0], { ...state[1], visible: !!iframeCanBeVisible }]);
```

Comment:
It isn't a big deal, but is it okay to name it a bit differently like:
```
setIFrameState(({ html, options }) => [html, { ...options, visible: !!iframeCanBeVisible }]
```

Reply:
Updated.